request.go

```diff
@@ -21,6 +22,16 @@ func (r *Request) Get(url string) (*Response, error) {
 	return r.client.doRequest(req)
 }
 
+// Post performs a request using the POST method
+func (r *Request) Post(url string, body io.Reader) (*Response, error) {
+	req, err := r.client.newRequest(r.context, url, http.MethodPost, nil)
```

Comment:
I noticed that the received body is not being used. Please specify when creating the request:
```go
req, err := r.client.newRequest(r.context, url, http.MethodPost, body)
```